packages/slice-machine/src/features/onboarding/OnboardingGuide.module.css

```diff
@@ -0,0 +1,14 @@
+.onboardingCard {
```

Comment:
💡 #idea: I think you could use Card component from the `editor-ui`

Reply:
Yeah sounds good, applied 👍